rasterio/__init__.py

```diff
@@ -32,11 +32,14 @@ def emit(self, record):
     DatasetReader, get_writer_for_path, get_writer_for_driver, MemoryFile)
 from rasterio.profiles import default_gtiff_profile
 from rasterio.transform import Affine, guard_transform
-from rasterio.vfs import parse_path
+from rasterio.path import parse_path
 
 # These modules are imported from the Cython extensions, but are also import
 # here to help tools like cx_Freeze find them automatically
-from rasterio import _err, coords, enums, vfs
+import rasterio._err
+import rasterio.coords
+import rasterio.enums
+import rasterio.path
```

Comment:
TODO: remove these in another PR? It's no longer a complete list of modules that freezing tools would like to have and nobody seems to notice or care.